doc/whats_new.rst

```diff
@@ -15,14 +15,17 @@ Current
 Changelog
 ~~~~~~~~~
 
-- Add ability to simulate multiple trials in parallel using joblibs, by `Mainak Jas`_ in `#44 <https://github.com/jasmainak/hnn-core/pull/44>`_
+- Add ability to simulate multiple trials in parallel using joblibs, by `Mainak Jas`_ in `#44 <https://github.com/hnnsolver/hnn-core/pull/44>`_
+- Support loading of legacy .param files, by `Blake Caldwell`_ in `#80 <https://github.com/hnnsolver/hnn-core/pull/80>`_
 
 Bug
 ~~~
 
-- Fix missing autapses in network construction, by `Mainak Jas`_ in `#50 <https://github.com/jasmainak/hnn-core/pull/50>`_
+- Fix missing autapses in network construction, by `Mainak Jas`_ in `#50 <https://github.com/hnnsolver/hnn-core/pull/50>`_
 
 API
 ~~~
 
+- Changed from Params(fname) to Params(param_dict) with addition of read_params(fname) in `#80 <https://github.com/hnnsolver/hnn-core/pull/80>`_
```

Comment:
Sorry for being nitpicky, additions go into "Changelog"

Reply:
So the change would remain here in API and the addition would go to the changelog

Reply:
Here:
https://github.com/hnnsolver/hnn-core/pull/80/files#diff-3433b590ca8611f80d7687b51a869383R19